deno.jsonc

```diff
@@ -0,0 +1,38 @@
+{
+	// https://deno.land/manual/getting_started/configuration_file
+	"lock": false,
+	// https://deno.land/manual/advanced/typescript/configuration
+	// https://www.typescriptlang.org/tsconfig/
+	"compilerOptions": {
+		"noFallthroughCasesInSwitch": true,
+		"noImplicitOverride": true,
+		"noImplicitReturns": true,
+		"noPropertyAccessFromIndexSignature": true,
+		"noUncheckedIndexedAccess": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"useUnknownInCatchVariables": true
+	},
```

Comment:
Deno discourages `"compilerOptions"`.

Reply:
Then they should use better ones per default? I do prefer more pedantic checking of mistakes and Deno lint is not perfect. Typescript can find more mistakes compared to not using it. Do you have other ideas how to achieve that without Typescript?

Reply:
Options not covered by the linter could be upstreamed to https://github.com/grammyjs/grammY/pull/469. It's high time to finish the PR @KnorpelSenf.

Reply:
Yeah ........................